AffirmSDK/AffirmAsLowAsButton.h

```diff
@@ -14,6 +14,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface AffirmAsLowAsButton : UIButton
 
 @property (nonatomic, weak) id presentingViewController;
+@property (nonatomic, strong) NSString *promoID;
```

Comment:
This is a change to the SDK and not the demo app. Can we keep this PR only for the demo app and make this change in another PR if required?

Reply:
Done—see #34